### PR TITLE
On branch main

### PR DESCRIPTION
**Pull request by Bader-ed**
Your branch is up to date with 'origin/main'.

Changes to be committed:
	modified:   package.json
	new file:   src/app/(about us)/about/page.tsx
	modified:   src/app/(landingPage)/_components/footer.tsx
	modified:   src/app/(main)/_components/CNavbar.tsx
	modified:   src/app/(main)/_components/Sidebar.tsx

### CODE REVIEW

**Review by chikobara — CHANGES_REQUESTED** [review]
make the about page that you redirect user for

**Review by chikobara — APPROVED** [review]
There must be a huge work on the About page, 
Approved for now

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "lucide-react": "^0.412.0",
     "next": "14.2.5",
     "next-pwa": "^5.6.0",
+    "next-share": "^0.27.0",
```
Comment (chikobara) [inline package.json:43]: whats this for  ???

File: src/app/about/page.tsx
Comment (chikobara) [inline src/app/about/page.tsx:1]: also you should move it to `(about us)/about`
and change the `(about us)` to `(about)`

**Comment by Bader-ed** [comment]
Package for next-share to make the social share button in the document page

File: package.json (hunk 1 above)
Comment (Bader-ed) [reply package.json:43]: Package for next-share to make the social share button in the document page

**Pull request closed without merging**